### ISSUE TITLE
The texture of the glTF 2.0 model in the repository is broken

### ISSUE THREAD

**Issue by cx20** [assimp/assimp#2204]
It seems that the texture of the glTF file in the following folder is corrupted.
https://github.com/assimp/assimp/tree/master/test/models/glTF2/BoxTextured-glTF
https://github.com/assimp/assimp/tree/master/test/models/glTF2/BoxTextured-glTF-Binary
https://github.com/assimp/assimp/tree/master/test/models/glTF2/BoxTextured-glTF-Embedded
https://github.com/assimp/assimp/tree/master/test/models/glTF2/BoxTextured-glTF-pbrSpecularGlossiness

I think it would be better to update with Khronos' glTF Sample model.
https://github.com/KhronosGroup/glTF-Sample-Models/tree/master/2.0/BoxTextured/glTF
https://github.com/KhronosGroup/glTF-Sample-Models/tree/master/2.0/BoxTextured/glTF-Binary
https://github.com/KhronosGroup/glTF-Sample-Models/tree/master/2.0/BoxTextured/glTF-Embedded
https://github.com/KhronosGroup/glTF-Sample-Models/tree/master/2.0/BoxTextured/glTF-pbrSpecularGlossiness

**Comment by kimkulling** [comment]
Yes, was corrupted by one reformat commit. Thanks for figuring this out. I will take care.

**Comment by aavenel** [comment]
Looks like this is this commit that broke everything : a8a1ca9894a38164f3111f57fddb887b3549ea60
